flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/FlinkStreamRuleSets.scala

```diff
@@ -109,6 +109,8 @@ object FlinkStreamRuleSets {
       List(
         //removes constant keys from an Agg
         AggregateProjectPullUpConstantsRule.INSTANCE,
+        //fix: FLINK-17553 unsupported call error when constant exists in group window key
+        ProjectMergeRule.INSTANCE,
```

Comment:
add the following sentence: this rule will merge the project generated by `AggregateProjectPullUpConstantsRule` and make sure window aggregate can be correctly rewritten by StreamLogicalWindowAggregateRule